src/schema.py

```diff
@@ -1,46 +1,57 @@
 from datetime import datetime
 from dateutil import parser
 
-from graphene import Boolean, Field, Int, List, ObjectType, String
+from graphene import Field, Int, List, ObjectType, String
 from graphene.types.datetime import Date
 import pytz
 import requests
 
 from src.constants import *
 from src.types import (
     AccountInfoType,
-    EateryType,
+    CampusEateryType,
+    CollegetownEateryType,
     TransactionType,
 )
 
 class Data(object):
-  eateries = {}
+  campus_eateries = {}
+  collegetown_eateries = {}
+
 
   @staticmethod
-  def update_data(eateries):
-    Data.eateries = eateries
+  def update_data(campus_eateries, collegetown_eateries):
+    Data.campus_eateries = campus_eateries
+    Data.collegetown_eateries = collegetown_eateries
 
 class Query(ObjectType):
-  eateries = List(
-      EateryType,
+  campus_eateries = List(
+      CampusEateryType,
+      eatery_id=Int(name='id'),
+  )
+  collegetown_eateries = List(
+      CollegetownEateryType,
       eatery_id=Int(name='id'),
-      date=Date(),
-      eatery_name=String(name='name'),
-      campus_area=String(name='area'),
-      is_open=Boolean()
   )
   account_info = Field(
       AccountInfoType,
       date=Date(),
       session_id=String(name='id')
   )
 
-  def resolve_eateries(self, info, eatery_id=None):
+  def get_eateries(eateries, eatery_id):
     if eatery_id is None:
-      return [eatery for eatery in Data.eateries.values()]
-    eatery = Data.eateries.get(eatery_id)
+      return [eatery for eatery in eateries.values()]
```

Comment:
`list(eateries.values())` should work

Reply:
ah very clever, can we make this change in a PR -> master @cswenberg?

Reply:
on it